### PR TITLE
Brew auth is default to not be set

### PR DESCRIPTION
**Pull request by oharan2**
Brew token is not mandatory for rhods addon since we pick released version from the API.
For version 1.29.0 the fact we set it caused failures.

### CODE REVIEW

File: ocm_python_wrapper/cluster.py [hunk 1]
```diff
@@ -595,7 +595,10 @@ def create_rhods_brew_config(brew_token):
                 }
             ],
         )
-        secret_data_dict = {"auths": {"brew.registry.redhat.io": {"auth": brew_token}}}
```
Comment (oharan2) [inline ocm_python_wrapper/cluster.py:598]: `auth` key with None value is failing, therefore, the dict should be empty if `brew_token=None`.

File: ocm_python_wrapper/cluster.py [hunk 2]
```diff
@@ -595,7 +595,10 @@ def create_rhods_brew_config(brew_token):
...
+
+        secret_data_dict = {"auths": {"brew.registry.redhat.io": {}}}
```
Comment (rnetser) [inline ocm_python_wrapper/cluster.py:599]: why is this needed when there's no token?

**Comment by oharan2** [comment]
Wont do
